src/2023/talks/index.html

```diff
@@ -0,0 +1,287 @@
+                </p>
+              </div>
+            </section>
+
+            <section class="stack">
+              <h3>
+                How to Keep your Database Happy
+              </h3>
+
+              <p>
+                You don't want to spend too much time looking after your database; you've got better things to do with your time, but you do want your database to run smoothly and perform well. Fortunately, there are a few simple things that you can do to make sure your database ticks along nicely in the background.
+              </p>
+              <p>
+                I've put together my top 5 tips, based on things that have been useful to me as a DBA. The focus will be on Postgres, but most of the tips are also relevant to other databases. These are things you can put in place, without too much effort, to make sure your database works well.
+              </p>
+            </section>
+          </article>
+
+          <article class="stack region" id="raia-anatomy-of-a-sonic-pi-song">
+            <section class="stack">
+              <div class="stack speaker-preview">
+                <figure>
+                  <img class="speaker-picture" width="200" height="200" alt="Placeholder speaker picture for Raia" src="../images/speakers/placeholder.png">
+                </figure>
+                <h2>
+                  Raia
+                </h2>
+              </div>
+
+              <div class="stack">
+                <p>
+                  Since 2015, Raia has been creating original live coded compositions in Sonic Pi with vocal layering. A founding member of Algorave India, much of their live coding is shared through workshops and live shows, supported by zkm, Goethe institute and the Canada Council for Arts. Being autodidactic in music and coding, xe brings a unique outlook to live coding production. The resulting effects are lyrical music, enveloped by code, with the potential to connect.
+                </p>
+              </div>
+            </section>
+
+            <section class="stack">
+              <h3>
+                Anatomy of a Sonic Pi Song
+              </h3>
+
+              <p>
+                Have you ever considered what makes a "good" song? Maybe it's a sweet melody, or the beat that keeps you pushing through to the end of a workout. Whatever the use case, Ruby-based Sonic Pi can synthesize it. Raia guides participants through coding a range of song components. Together, we'll build a band of live loops including: rhythm sections, melody, chord progressions and audio embellishments. We'll build a song that can be mixed and modified at will and in the process, learn some key elements of what makes Sonic Pi "sing.”
+              </p>
+            </section>
+          </article>
+
+        </div>
+      </main>
+      <footer></footer>
```

Comment:
I'd expect to have a footer?

Reply:
I wasn't sure what to put in there yet, so I left it blank. (If I remove, I'd also need to tweak the layout which is currently managed as a [`cover` ](https://every-layout.dev/layouts/cover/))